lib/Controller/RoomController.php

```diff
@@ -1907,6 +1910,50 @@ public function verifyDialInPin(string $pin): DataResponse {
 		return new DataResponse($this->formatRoom($this->room, $participant, skipLastMessage: true));
 	}
 
+	/**
+	 * Direct dial-in (SIP bridge)
+	 *
+	 * @param string $phoneNumber Phone number that is called
+	 * @return DataResponse<Http::STATUS_OK, TalkRoom, array{}>|DataResponse<Http::STATUS_UNAUTHORIZED|Http::STATUS_NOT_FOUND|Http::STATUS_NOT_IMPLEMENTED, null, array{}>
+	 *
+	 * 200: Participant returned
+	 * 401: SIP request invalid
+	 * 404: Participant not found
+	 * 501: SIP dial-in is not configured
+	 */
+	#[PublicPage]
+	#[BruteForceProtection(action: 'talkSipBridgeSecret')]
+	#[OpenAPI(scope: 'backend-sipbridge')]
+	#[RequireRoom]
+	public function directDialIn(string $phoneNumber): DataResponse {
```

Comment:
This should also receive the number of the caller so the rooms could be returned depending on who is calling.

Reply:
@fancycode when someone calls with masked number, you send us empty string? or null, or …?
Since the current take is "Reuse room for same number" I kind of need to know when not to look but create a new conversation instead.

Reply:
Well, problem is there is no clear definition for that case in SIP. Some providers send "Anonymous" / "anonymous", some send an empty number, etc.

Maybe in a first step you can handle all non-numeric numbers as "anonymous"? In the future there might be cases where this should be supported (e.g. direct calling from "known" devices where the SIP provider might send the name of the user that is calling as display name in the call, e.g. `"John Doe" <sip:1234@asterisk.sipprovider.invalid>`).

Reply:
Okay, so you don't get to parse that info before, or you don't want to?
We can also `lowercase and in array['anonymous']` with the array being a config administrators can extend, based on user feedback?